website/Makefile

```diff
@@ -40,6 +40,7 @@ public/index.html:
 clean-sphinx-site:
 	rm -rf sphinx/_build
 	rm -rf sphinx/api
+	git restore sphinx/api/mlos.rst
```

Comment:
Kind of a hack, but it works ...

Reply:
I mean the whole sphinx business is a bit hacky right now and I take all the blame :-/

Reply:
I added to it, so I think we can share that blame :)